src/assembler/parser/text.ts

```diff
@@ -52,6 +54,20 @@ export function parseText(text: string, rules: ITextParseRule[]): ITextCursor {
                 ? match[0].length - match[0].lastIndexOf('\n') - 1
                 : startCursor.position + match[0].length
           }
+          if (rule.name == 'Instruction') {
+            let charCodeBeforeInstruction = text.charCodeAt(
+              startCursor.index - 1
+            )
+            if (
+              charCodeBeforeInstruction !== 32 &&
```

Comment:
Some magic numbers. Add a comment or make them as a "real" char and compare it to the char from the text.

Reply:
made it with real chars